www/contribute/producing-an-ebook-step-by-step.php

```diff
@@ -448,8 +450,11 @@
 						<code class="terminal"><span><b>se</b> interactive-replace <i>"([Mm])ean time" "\1eantime"</i> src/epub/text/<i class="glob">*</i></span></code>
 					</li>
 				</ul>
-				<p>After you’ve reviewed the changes, create an <code class="html">[Editorial]</code> commit. This commit is important, because it gives purists an avenue to revert modernizing changes to the original text.</p>
-				<p>Note how we preface this commit with <code class="html">[Editorial]</code>. Any change you make to the source text that can be considered a modernization or editorial change should be prefaced like this, so that the <code class="bash"><b>git</b></code> history can be easily searched by people looking to revert changes.</p>
+				<p>After you’ve reviewed the changes, create an <code class="html">[Editorial]</code> commit. This type of commit is important, because it gives purists an avenue to reverse these changes back to the original text.</p>
+				<aside class="alert">
+					<p>Editorial changes are those where we make an editorial decision to alter the original text, for example modernizing spelling or fixing a probable printer’s typo; but not fixing a transcriber’s typo.</p>
+					<p>Don’t use the [Editorial] flag for typographical changes such as hyphenation or for changing the case of letters or words.</p>
```

Comment:
This is not quite right, they should do that, since modernize-spelling adjusts hyphenation